specs/Modal.spec.js

```diff
@@ -234,25 +244,42 @@ describe('Modal', () => {
     const modal = renderModal({ isOpen: true });
     expect(modal.portal.content.style.position).toEqual(newStyle);
     Modal.defaultStyles.content.position = previousStyle;
+    unmountModal();
+  });
+
+  it('should remove class from body when no modals opened', () => {
+    const expectedBodyClass = 'ReactModal__Body--open';
+    const classList = document.body.classList;
+    renderModal({ isOpen: true });
+    renderModal({ isOpen: true });
+    expect(classList.contains(expectedBodyClass)).toBe(true);
+    unmountModal();
+    expect(classList.contains(expectedBodyClass)).toBe(false);
```

Comment:
If using redux or a normal component state, you can set an state to avoid opening more modals at the same time. 

Just to illustrate the idea:

```javascript
{ currentModal: null } // can open
```

```javascript
{ currentModal: 'my-modal-1' } // already in use
```